deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4630,6 +4615,22 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       info.GetReturnValue().Set(JS_BOOL(gl->premultiplyAlpha));
       break;
     }
+    case MAX_CLIENT_WAIT_TIMEOUT_WEBGL:
+    case GL_MAX_SERVER_WAIT_TIMEOUT:
+    {
```

Comment:
These were returning -1 in exokit but 0 in Chrome. Since they're timeouts, is it safe to punt here and always return 0?

Reply:
Curious if this was because of a legitimate GL error...